For-Loop/05.LeftAndRightSum/05.LeftAndRightSum.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace _05.LeftAndRightSum
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int n = int.Parse(Console.ReadLine());
+
+            int numNumbers = 2 * n;
+            int leftSum = 0;
+            int rightSum = 0;
+
+            for (int i = 0; i < numNumbers; i++)
+            {
+                int num = int.Parse(Console.ReadLine());
+
+                if (i < n)
```

Comment:
It is not good practice to check iterator anywhere except loop definition (line 15). I would do two separate for loops (you can extract them as a method in order not to repeat yourself).